src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -216,6 +231,38 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     return this.applyFilters(filterItems, allItems);
   }
 
+  updateColumnFilter(checkboxValue: boolean, columnName: string) {
+    if (checkboxValue){
+      this.hiddenColumnNames.delete(columnName);
+    } else {
+      this.hiddenColumnNames.add(columnName);
+    }
+  }
+
+  showColumn(index: number): boolean {
```

Comment:
Nothing critical and maybe it's just me, but maybe `showColumn` method could have another name since it feels like it will render something without returning any value. @Nokel81 wdyt?

Reply:
`columnIsVisible`?

Reply:
fixed